README.md

```diff
@@ -85,6 +85,9 @@ $ ibmcloud ce project create --name demos --select
 - [ping](ping)<br>
   This will show how to setup a simple Ping(cron) Event Source and send
   its events to an Application.
+- [cos-event](cos-event)<br>
```

Comment:
I think you need to add the 'cos' plugin to the list at line 31

Reply:
Done.